### PR TITLE
Login UI (ChicoState#3)

### PR DESCRIPTION
**Pull request by asheelamagwili**
Basic login UI implemented in _lib/auth/login.dart_ , followed general styling direction by @bmarden . This file also contains comments of my current test codes for Google Authentication. Should be able to run it and see the login UI if:

`home: LoginPage(title: 'Login'),`

is uncommented in _main.dart_ and 

`home: HomePage(title: 'Study Buddy App'), // Placeholder title for now`

is commented out.

### CODE REVIEW

**Review by lrondenet — COMMENTED** [review]
Looks Good!

**Review by lrondenet — APPROVED** [review]

**Review by e7ite — APPROVED** [review]
Freakin' awesome job!

File: study_group_app/lib/screens/auth/login.dart [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  Widget build(BuildContext context) {
+    // Scaffold is from MaterialApp. Implements the basic visual layout
+    return Scaffold(
+      appBar: AppBar(
+        // Top bar, title is set using widget.title which is passed down from main
+        title: Text(widget.title),
+      ),
+      // Container: UI for Login Form
+      body: Container(
+        padding: EdgeInsets.fromLTRB(20, 300, 20, 40),
```
Comment (bmarden) [inline study_group_app/lib/screens/auth/login.dart]: I'm getting bottom overflow from this when the keyboard is brought up. Could change it to this to fix that issue and then we can reposition the text boxes later if we want. 
```suggestion
        padding: EdgeInsets.all(20),
```

**Review by bmarden — CHANGES_REQUESTED** [review]

**Review by bmarden — APPROVED** [review]
Looks good

**Pull request merged**